packages/fx-core/src/component/driver/file/updateEnv.ts

```diff
@@ -30,7 +31,10 @@ const helpLink = "https://aka.ms/teamsfx-actions/file-updateEnv";
 export class UpdateEnvDriver implements StepDriver {
   description = getLocalizedString("driver.file.description");
 
-  @hooks([addStartAndEndTelemetry(actionName, actionName)])
+  @hooks([
+    addStartAndEndTelemetry(actionName, actionName),
+    updateProgress(getLocalizedString("driver.file.progressBar.env")),
+  ])
```

Comment:
Can we totally remove `file/updateEnv` and `file/updateJson` since they are deprecated?

Reply:
Will do in another PR.
Some preview samples or vs projects still use the old ones.